OSDOCS#9040: Adding note warning of future PSA restricted enforcement (#69218)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

//[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
//=== Cloud controller managers for additional cloud providers

[discrete]
[id="ocp-4-15-planned-psa-restricted-enforcement"]
=== Future restricted enforcement for pod security admission

Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.

> Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.

Reviewer: In general, I thought we weren't supposed to mention features of future releases in the docs[1]. I assume there's a justifiable reason for including this info in the docs, though.

[1] https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Author: Yes, this one is okay because it is a warning to prepare. It has the proper fluffy warning that allows for flexibility if the plans change (which they have done, several times). Thanks for checking!

> To prepare for this upcoming change, ensure that your workloads match the pod security admission profile that applies to them. Workloads that are not configured according to the enforced security standards defined globally or at the namespace level will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.

Reviewer: ```suggestion
To prepare for this upcoming change, ensure that your workloads match the pod security admission profile that applies to them. If your workloads are not configured according to the enforced security standards defined globally or at the namespace level, they will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
```
Technically, your original sentence is correct, but I thought I'd offer a suggestion to make it a little more engaging by directly addressing the reader, as you did in the first sentence.

For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Cluster Operators reference_.

[id="ocp-4-15-planned-psa-restricted-enforcement"]
=== Future restricted enforcement for pod security admission

> Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.

Reviewer 2: ```suggestion
Currently, pod security violations are shown as warnings in the audit logs without resulting in the rejection of the pod.
```

Author: Works for me! Updated - please take another look and let me know how it looks, thanks!
